sda-commons-server-dropwizard/src/test/java/org/sdase/commons/server/dropwizard/bundles/scanner/JacksonTypeScannerTest.java

```diff
@@ -126,6 +128,7 @@ void shouldFindPossibleEnvs() {
         .contains("LISTOFLISTS_<INDEX>_<INDEX> (Array)")
         .contains("LISTOFMAPS_<INDEX>_<KEY> (Map)")
         .contains("LOGGING_LEVEL (String)")
+        .contains("LOGGING_LOGGERS_<KEY>_<ANY> (JsonNode)")
```

Comment:
Don't we expect a `String` value or even more precise a log level as value instead of `JsonNode`?
At least it would be more helpful for our customers to know what value they should use if they want to change the log level.

Edit: I now found the documentation about the "omitted" scanario to set a `String`. I find that hard to understand from the reader's / customer's perspective. Should we give a concrete example (for logging) to make it easier to understand?

Reply:
> Don't we expect a String value or even more precise a log level as value instead of JsonNode?

As said in the last discussion, Dropwizard is very flexible in that area. Imho it would get really messy, when we add exception in the generic code for a specific field.

I don't know, why someone should do it, but you can set a very detailed [LoggingConfiguration](https://github.com/dropwizard/dropwizard/blob/release/4.0.x/dropwizard-logging/src/main/java/io/dropwizard/logging/common/LoggerConfiguration.java) for a specific logger there as well.

> Should we give a concrete example …

To me it feels wrong to provide specific examples that expect that concrete properties are discovered for a generic discovery.

Reply:
But still I think it is a very common use case for our customers to change the log level. We could add information to the [documentation of the logging bundle](https://github.com/SDA-SE/sda-dropwizard-commons/blob/95e354f449ad0bba6e10f01610e542492b2ff7eb/docs/server-dropwizard.md?plain=1#L70) how this can be done in an easy way.

In the past I've pointed them to the [official Dropwizard documentation](https://www.dropwizard.io/en/latest/manual/core.html#configuration) that includes some examples how to override the log level. I think it is very helpful to have another central place where this behaviour is documented.

Reply:
What do you think about [this implementation](https://github.com/SDA-SE/sda-dropwizard-commons/pull/2913/commits/df9c1fdfc47d8598335934302e342260ba939004)?

Reply:
It does not really show how the JsonNode configuration works but just shows the most simple example of a logger configuration.

Reply:
Yes, I like it. It already helps a lot! 👍